deploy/sre-prometheus/100-excessive-memory.PrometheusRule.yaml

```
@@ -0,0 +1,28 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-excessive-memory
+    role: alert-rules
+  name: sre-excessive-memory
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-excessive-memory
+    rules:
+    - alert: ExcessiveContainerMemoryWarningSRE
+      expr: container_memory_rss{namespace=~"(^openshift.*|^kube.*|^default$)",container_name!="",container_name!="POD"}/1024/1024/1024>2
```

```suggestion
      expr: container_memory_rss{namespace=~"(^openshift.*|^kube.*|^default$)",container_name!="",container_name!="POD"}/1024/1024/1024>3
```

CatalogSource pods consume roughly 2.5GB, need this to be above that threshold.